src/OpenLoco/src/Game.cpp

```diff
@@ -47,7 +47,10 @@ namespace OpenLoco::Game
 
     static bool openBrowsePrompt(StringId titleId, browse_type type, const char* filter)
     {
-        Audio::pauseSound();
+        if (!isTitleMode())
+        {
+            Audio::pauseSound();
```

Comment:
do we ever want to pause music when the browser window is open?

Reply:
In Locomotion music is paused when opening browser window in game. But now I realize that it is paused because the game is paused, not because the window is open. Then yes, I think we can remove it, but we have to pause music when game is paused. This is not working now if I remove `Audio::pauseSound();` in this comment.

Reply:
I was partially wrong. By removing it, the music is still stopped in `PromptSaveWindow::open()` while pressing load game, but it is not stopped when pressing saving game..

Reply:
I see, this is necessary because of the prompt window appearing and pausing the game before the actual load window does, but the prompt doesn't always show so we need it in both places.